seed.js

```diff
@@ -0,0 +1,134 @@
+const db = require('./database/index.js');
+const Promise = require('bluebird');
+const faker = require('faker');
+
+//create an array of random size, with random user_id's
+var randomUsers = () => {
+  var arr = new Array(Math.floor(Math.random() * 15));
+
+  for (var i = 0; i < arr.length; i++) {
+    arr[i] = Math.floor(Math.random() * 99 + 1);
+  }
+  return arr.join(' ').toString();
```

Comment:
not sure if you need the .toString() since it should become a string on the join?